querier_selects.go

```diff
@@ -215,3 +215,13 @@ func (q *Querier) FindByPrimaryKeyFrom(table Table, pk interface{}) (Record, err
 func (q *Querier) Reload(record Record) error {
 	return q.FindByPrimaryKeyTo(record, record.PKValue())
 }
+
+// Count is a handy method for counting number of rows for a query
+func (q *Querier) Count(view View, tail string, args ...interface{}) (int, error) {
+	query := q.startQuery("SELECT COUNT(*) AS cnt FROM %s %s")
```

Comment:
`startQuery` should contain only `"SELECT"`, everything else should be added to the result.